packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -46,7 +57,9 @@ async function resolveShortname(): Promise<void> {
     return;
   }
   if (imageToPull && !imageToPull.includes('/')) {
-    shortnameImages = (await window.resolveShortnameImage(selectedProviderConnection, imageToPull)) ?? [];
+    shortnameImages =
+      (await window.resolveShortnameImage($state.snapshot(selectedProviderConnection), $state.snapshot(imageToPull))) ??
```

Comment:
question: new pattern to follow ?

Reply:
If we don't use `$state.snapshot` we get the proxy which cannot be serialized, we could maybe change the proxy logic, but I don't know how


We already use it some places

https://github.com/podman-desktop/podman-desktop/blob/4cd79ac55b66db0229f75e5b99304cfc236630ae/packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.svelte#L52